src/Utils/Position.ml

```diff
@@ -96,3 +96,106 @@ let to_string pos =
       pos.pos_fname
       pos.pos_start_line (start_column pos)
       pos.pos_end_line   (end_column pos)
+
+module PrettyPrinting =
+struct
+  type underline_options =
+    | NoUnderline
+    | UnderlineBegining
+    | UnderlineIfOneLine
+    | UnderlineAlways
+
+  type options = {
+    context : int;
+    underline : t -> underline_options;
+    add_line_numbers : bool;
+  }
+
+  let default_options = {
+    context = 1;
+    underline = (fun _ -> UnderlineIfOneLine);
+    add_line_numbers = true;
+  }
+
+end
+
+(** Get text from file from given position *)
+let get_text_range ?(options = PrettyPrinting.default_options) ?channel (pos : t) =
```

Comment:
Currently, the pretty printer uses only default options, right? If so, what are the future plans for other possible options?